fastchat/constants.py

```diff
@@ -16,6 +16,9 @@
 INACTIVE_MSG = "THIS SESSION HAS BEEN INACTIVE FOR TOO LONG. PLEASE REFRESH THIS PAGE."
 # Maximum input length
 INPUT_CHAR_LEN_LIMIT = int(os.getenv("FASTCHAT_INPUT_CHAR_LEN_LIMIT", 3072))
+INPUT_CHAR_LEN_LIMIT_WITH_IMAGE = int(
```

Comment:
why cannot we just reuse `INPUT_CHAR_LEN_LIMIT`

Reply:
I was following the convention from Llava but perhaps this is not needed